<ui version="4.0" >
 <class>PrinterPage</class>
 <widget class="QWidget" name="PrinterPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>568</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="headerLabel" >
     <property name="text" >
      <string>&lt;b>Configure printer settings&lt;/b></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line7" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="1" column="1" colspan="2" >
      <widget class="QFrame" name="frame_2" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="printerColorButton" >
          <property name="text" >
           <string>Colour</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="printerGrayscaleButton" >
          <property name="text" >
           <string>Gray Scale</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="printFirstPageFirstButton" >
          <property name="text" >
           <string>First Page First</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="printFirstPageLastButton" >
          <property name="text" >
           <string>Last Page First</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLineEdit" name="printerNameEdit" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel1" >
       <property name="text" >
        <string>Printername:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="TextLabel3" >
       <property name="text" >
        <string>Magnification:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2" >
      <widget class="QLineEdit" name="printheaderFontSample" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Header Font</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignHCenter</set>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QPushButton" name="printheaderFontButton" >
       <property name="toolTip" >
        <string>Press to select the font for the page headers</string>
       </property>
       <property name="text" >
        <string>Header Font</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QSpinBox" name="printMagnificationSpinBox" >
       <property name="minimum" >
        <number>-10</number>
       </property>
       <property name="maximum" >
        <number>10</number>
       </property>
       <property name="value" >
        <number>-3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="TextLabel2" >
       <property name="text" >
        <string>Colour Mode:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="TextLabel4" >
       <property name="text" >
        <string>Page Order:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>251</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Margins</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="1" >
         <widget class="QDoubleSpinBox" name="topMarginSpinBox" >
          <property name="toolTip" >
           <string>Enter the top margin in cm.</string>
          </property>
          <property name="suffix" >
           <string> cm</string>
          </property>
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>9.900000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QDoubleSpinBox" name="leftMarginSpinBox" >
          <property name="toolTip" >
           <string>Enter the left margin in cm.</string>
          </property>
          <property name="suffix" >
           <string> cm</string>
          </property>
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>9.900000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QDoubleSpinBox" name="rightMarginSpinBox" >
          <property name="toolTip" >
           <string>Enter the right margin in cm.</string>
          </property>
          <property name="suffix" >
           <string> cm</string>
          </property>
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>9.900000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QDoubleSpinBox" name="bottomMarginSpinBox" >
          <property name="toolTip" >
           <string>Enter the bottom margin in cm.</string>
          </property>
          <property name="suffix" >
           <string> cm</string>
          </property>
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>9.900000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>428</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>printerNameEdit</tabstop>
  <tabstop>printerColorButton</tabstop>
  <tabstop>printerGrayscaleButton</tabstop>
  <tabstop>printFirstPageFirstButton</tabstop>
  <tabstop>printFirstPageLastButton</tabstop>
  <tabstop>printMagnificationSpinBox</tabstop>
  <tabstop>printheaderFontButton</tabstop>
  <tabstop>topMarginSpinBox</tabstop>
  <tabstop>leftMarginSpinBox</tabstop>
  <tabstop>rightMarginSpinBox</tabstop>
  <tabstop>bottomMarginSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
